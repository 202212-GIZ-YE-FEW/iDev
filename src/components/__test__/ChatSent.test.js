import renderer from "react-test-renderer";

import Sent from "../ChatSent";

it("renders correctly", () => {
    const tree = renderer.create(<Sent />).toJSON();
    expect(tree).toMatchSnapshot();
});
