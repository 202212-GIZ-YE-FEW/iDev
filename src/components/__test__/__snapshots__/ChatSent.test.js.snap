// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-2/3 self-start p-3 rounded-lg"
>
  <div
    className="flex items-center"
  >
    <div
      className="inline-flex items-center justify-center h-10 w-10 rounded-full bg-light-gray flex-shrink-0"
    />
    <div
      className="relative ms-3 text-sm bg-light-cyan py-2 px-4 shadow rounded-xl"
    >
      <div />
    </div>
  </div>
</div>
`;
